<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto write form IRS - Trần Đức Tình</string>
  </property>
  <widget class="QPushButton" name="bntChooseFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>chọn file</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLinkfile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>chưa chọn file excell nào</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bntAction">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Bắt đầu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
